src/lib/lore.sh

```diff
@@ -450,9 +454,11 @@ function process_patchsets()
     esac
 
     ((count++))
-
   done <<< "$pre_processed_patches"
-  wait
+
+  for pid in "${pids[@]}"; do
+    wait "$pid"
+  done
```

Comment:
Maybe add a succinct comment to explain the why we need this for loop (think it's to not conflict with other background processes created outside this function, right?).